test/integration/auth/authz_config_test.go

```diff
@@ -63,6 +65,16 @@ authorizers:
 	)
 	t.Cleanup(server.TearDownFn)
 
+	// Make sure anonymous requests work
+	anonymousClient := clientset.NewForConfigOrDie(rest.AnonymousClientConfig(server.ClientConfig))
+	healthzResult, err := anonymousClient.DiscoveryClient.RESTClient().Get().AbsPath("/healthz").Do(context.TODO()).Raw()
+	if !bytes.Equal(healthzResult, []byte(`ok`)) {
+		t.Fatalf("expected 'ok', got %s", string(healthzResult))
```

Comment:
anonymous request will hit this https://github.com/kubernetes/kubernetes/pull/121818/files#diff-15060c795e56fc5b2cc6584cac817d93dc0993f46f6b8eb3ecb57b77d6cd830aR92-R94 code path right? is there a way to test `authzmodes.ModeAlwaysAllow`

Reply:
AlwaysAllow is tested in a bunch of places already

the integration test bit I added requires *both* anonymous auth and RBAC default policy to be working in order to work, so it exercises both fixes at once